client/my-sites/plans/plan-overview/plan-status/plan-status-progress/index.jsx

```diff
@@ -37,26 +31,32 @@ const PlanStatusProgress = React.createClass( {
 	},
 
 	renderDaysRemaining() {
+		const { plan } = this.props;
+
 		return this.translate(
 			'%(daysUntilExpiry)s day remaining',
 			'%(daysUntilExpiry)s days remaining',
```

Comment:
If the `daysUntilExpiry` argument is always an integer we could reference it as `%(daysUntilExpiry)d` instead.